chain/rosetta-rpc/src/models.rs

```diff
@@ -719,15 +719,18 @@ pub(crate) enum OperationType {
     Apiv2Schema,
     strum::EnumIter,
 )]
-#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
 pub(crate) enum OperationStatusKind {
+    #[serde(rename = "SCREAMING_SNAKE_CASE")]
     Success,
+    #[serde(rename = "")]
```

Comment:
Please, add a comment explaining the context why we introduced this status instead of following the spec (ideally, we need some point of contact who requested this change referenced here if possible)